docs/tutorials/react-native.mdx

```diff
@@ -0,0 +1,672 @@
+    <View style={styles.container}>
+      <View style={styles.videoContainer}>
+        // highlight-start
+        {peers.map((peer) =>
+          peer.tracks[0] ? (
+            <VideoRendererView
+              trackId={peer.tracks[0].id}
+              style={styles.video}
+            />
+          ) : null
+        )}
+        // highlight-end
+      </View>
+    </View>
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    alignItems: "center",
+    justifyContent: "space-between",
+    backgroundColor: "#F1FAFE",
+    padding: 24,
+  },
+  videoContainer: {
+    flexDirection: "row",
+    gap: 8,
+    flexWrap: "wrap",
+  },
+  video: { width: 200, height: 200 },
+});
+
+export default RoomScreen;
+```
+
+You should now see your own camera on your mobile device. You can add another
+participant and their new track (displaying for example rotating frog) in the dashboard like
+this:
+
+![Adding new participant](/img/tutorials/adding_new_track.gif)
+
+It should show up in the Room screen automatically:
+
+![Room screen](/img/tutorials/room_screen.png)
+
+For your convenience in our components library we provided a component to layout
+videos in a nice grid:
+
+```tsx title="/screens/Room.tsx"
+// highlight-next-line
+import { VideosGrid } from "@jellyfish-dev/react-native-jellyfish-components";
+
+function RoomScreen({ navigation }): JSX.Element {
+  const peers = usePeers();
+
+  return (
+    <View style={styles.container}>
+      // highlight-start
+      <VideosGrid
+        tracks={peers.map((peer) => peer.tracks[0]?.id).filter((t) => t)}
+      />
+      // highlight-end
+    </View>
+  );
+}
+```
+
+### Gracefully leaving the room
+
+To leave a room we'll add a button for the user. When the user clicks it, we
+gracefully leave the room, close the server connection and go back to the
+Connect screen.
+
+For leaving the room and closing the server connection you can use the `cleanUp` method from the `useJellyfishClient()` hook.
+
+After leaving the room we go back to the Connect screen.
+
+```tsx title="/screens/Room.tsx"
+// ...
+import {
+  usePeers,
+  VideoRendererView,
+  // highlight-next-line
+  useJellyfishClient,
+} from "@jellyfish-dev/react-native-client-sdk";
+// highlight-next-line
+import { InCallButton } from "@jellyfish-dev/react-native-jellyfish-components";
+
+function RoomScreen({ navigation }): JSX.Element {
+  const peers = usePeers();
+  // highlight-start
+  const { cleanUp } = useJellyfishClient();
+
+  const onDisconnectPress = () => {
+    cleanUp();
+    navigation.goBack();
+  };
+  // highlight-end
+
+  return (
+    <View style={styles.container}>
+      <VideosGrid
+        tracks={peers.map((peer) => peer.tracks[0]?.id).filter((t) => t)}
+      />
+      // highlight-start
+      <InCallButton
+        type="disconnect"
+        iconName="phone-hangup"
+        onPress={onDisconnectPress}
+      />
+      // highlight-end
+    </View>
+  );
+}
+
+// ...
+```
+
+## Summary
```

Comment:
And still don't work. This is my outcome:
https://github.com/kamil-stasiak/JellyfishReactNativeMinimalExample

Could you help me with that?

<img width="647" alt="image" src="https://github.com/jellyfish-dev/jellyfish-docs/assets/19818474/7afe3532-e27b-4dda-947f-a92cf0279e25">
<img width="386" alt="image" src="https://github.com/jellyfish-dev/jellyfish-docs/assets/19818474/9ed2727f-efc5-452a-a5c2-c49439771bd1">

<img width="388" alt="image" src="https://github.com/jellyfish-dev/jellyfish-docs/assets/19818474/0454d61a-0f35-418a-84ec-73cd6bf755df">